andela/resly
# Ft booking multiple tables 117327517

#### What does this PR do?

This PR enables users to book more than one table in a restaurant.
#### Description of Task to be completed?

Users can book a single table or hold as many tables as they want before making the booking.
#### How should this be manually tested?

> Visit a restaurant on Resly.
> Book one table by clicking on `Book Now`
> Click on `Hold` to hold more that one table.

The table status is changed to `Unhold`.

> Click on `Unhold` to unhold the table if it no longer one of the choices.
> 
> Click on `Process Bookings` to process the booking of multiple tables.

The tables are then added to the cart.
#### Any background context you want to provide?

Users previously could not make multiple bookings. They could only book a table. If there was a need to book more than one table, they would have to go through the same process for all the tables.
#### What are the relevant pivotal tracker stories?

This story can be viewed on Pivotal Tracker [here](https://www.pivotaltracker.com/projects/1415814/stories/117327517).
#### Screenshots (if appropriate)

![screen shot 2016-04-18 at 10 43 56 am](https://cloud.githubusercontent.com/assets/17028608/14600693/f063cd6e-0555-11e6-818f-c79308ba8084.png)
![screen shot 2016-04-18 at 10 44 15 am](https://cloud.githubusercontent.com/assets/17028608/14600694/f064dd9e-0555-11e6-8bab-12be70c4cded.png)
![screen shot 2016-04-18 at 10 44 36 am](https://cloud.githubusercontent.com/assets/17028608/14600696/f06547d4-0555-11e6-9ae9-f4c770d1d973.png)
![screen shot 2016-04-18 at 10 44 53 am](https://cloud.githubusercontent.com/assets/17028608/14600697/f066055c-0555-11e6-9728-a34f2b93e233.png)
![screen shot 2016-04-18 at 10 45 10 am](https://cloud.githubusercontent.com/assets/17028608/14600695/f0654414-0555-11e6-8f16-0c030e39d745.png)
![screen shot 2016-04-18 at 10 45 20 am](https://cloud.githubusercontent.com/assets/17028608/14600698/f0662d2a-0555-11e6-9295-ab21f5bbf1a2.png)
![screen shot 2016-04-18 at 10 45 31 am](https://cloud.githubusercontent.com/assets/17028608/14600700/f08512ee-0555-11e6-82fc-d208a0974cb0.png)
![screen shot 2016-04-18 at 10 45 51 am](https://cloud.githubusercontent.com/assets/17028608/14600699/f08500a6-0555-11e6-8508-9511c7d755e3.png)
![screen shot 2016-04-18 at 10 46 04 am](https://cloud.githubusercontent.com/assets/17028608/14600702/f08808e6-0555-11e6-8079-9a3429abd121.png)
![screen shot 2016-04-18 at 10 46 53 am](https://cloud.githubusercontent.com/assets/17028608/14600701/f08532a6-0555-11e6-97ee-6df1b82d36ff.png)
#### Question.

There are no questions at this moment.
